### ISSUE TITLE
Incorrect role as "Checkbox" is provided to "Primary sidebar", "Panel" and "Secondary sidebar" controls: A11y_Visual Studio Code Client_Customize Layout_DevTools

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#293807]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

### GitHub Tags: 
#A11yTCS;#Visual Studio Code Client;#GradeB_VisualStudioCodeClient_Win32_Feb2026;#DesktopApp;#FTP;#Win32;#A11ySev3;#A11yMAS;#WCAG4.1.2; #DevTools;

### Environmental Details: 
App Name: Visual Studio Code Client 
#URL: Visual Studio Code Insider
Microsoft Windows 11 Enterprise 
OS Build:  26200.7392

### Repro Steps: 
1. Open Visual Studio Code editor
2. TAB till "Primary sidebar" control.
3. Open accessibility insights for windows and inspect the control.
4. Observe the issue Incorrect role as "Checkbox" is provided to "Primary sidebar", "Panel" and "Secondary sidebar" controls

### Actual Results: 
Incorrect role as "Checkbox" is provided to "Primary sidebar", "Panel" and "Secondary sidebar" controls

**Note: Toggle string should be removed from name property of "Primary sidebar", "Panel" and "Secondary sidebar" controls.**

### Expected Results: 
Role as "Toggle button" should be provided to "Primary sidebar", "Panel" and "Secondary sidebar" controls

### User Impact: 
Screen reader only users will not be able to understand the correct purpose of the control.

### Attachments:

<img width="1364" height="697" alt="Image" src="https://github.com/user-attachments/assets/99b5641c-9040-4eab-ab78-26511299dba8" />
